staging/src/k8s.io/client-go/tools/cache/shared_informer_test.go

```diff
@@ -348,6 +348,18 @@ func TestSharedInformerWatchDisruption(t *testing.T) {
 	// Simulate a connection loss (or even just a too-old-watch)
 	source.ResetWatch()
 
+	// Wait long enough for the reflector to exit and the backoff function to start waiting
```

Comment:
just curios, what is "long enough" in this context? the time to execute all these actions?